pkg/controller/jobframework/reconciler.go

```diff
@@ -36,6 +36,7 @@ import (
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
 	"sigs.k8s.io/kueue/pkg/constants"
 	controllerconsts "sigs.k8s.io/kueue/pkg/controller/constants"
+	utilequality "sigs.k8s.io/kueue/pkg/util/equality"
```

Comment:
```suggestion
	"sigs.k8s.io/kueue/pkg/util/equality"
```

Unless there is a conflict